File: ipman.shared/Entity/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ipman.shared.Entity
{
    public class Person :  EntityBase
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime CreatedUTC { get; set; }
        public DateTime LastUpdatedUTC { get; set; }
        public bool IsActive { get; set; }
        public List<FaceCollection> FaceCollections { get; set; }
    }
}

File: ipman/ClientApp/entity/Person.ts
import { EntityBase } from "./EntityBase";
import { FaceCollection } from "@entity";

export default interface Person extends EntityBase
{
    FirstName: string;
    LastName: string;
    CreatedUTC: Date | string;
    LastUpdatedUTC: Date | string;
    IsActive: boolean;
    FaceCollections: FaceCollection[];
}